### PR TITLE
missing-patch-comment: look for inline comments

### LINKED ISSUE #109: missing-patch-comment doesn't look for inline comments
Example: https://github.com/NixOS/nixpkgs/pull/121005#issuecomment-828398589
It's looking only on the line above, not on the line containing the patch.

### PR DESCRIPTION
**Pull request by rmcgibbo**
Fixes #109

### CODE REVIEW

**Review by NinjaTrappeur — APPROVED** [review]
It does fix the linked issue though.

**Comment by rmcgibbo** [comment]
What kind of a monster would put a comment _before_ the patch inline though :)?

File: rust-checks/src/comment_finders.rs [hunk 1]
```diff
@@ -6,26 +6,40 @@ pub fn find_comment_within(node: &SyntaxNode) -> Option<String> {
...
     // Note: The `prev_siblings` iterator includes self, which
     // is not a `TOKEN_COMMENT`, so we need to skip one.
     let node_iter = prev_siblings(&NodeOrToken::Node(node.clone())).skip(1);
-    let doc = collect_comment_text(node_iter);
+    let doc = collect_comment_text(node_iter, true);
     Some(doc).filter(|it| !it.is_empty())
 }
 
-fn collect_comment_text(node_iter: impl Iterator<Item = SyntaxElement>) -> String {
+pub fn find_comment_after(node: &SyntaxNode) -> Option<String> {
```
Comment (jtojnar) [inline rust-checks/src/comment_finders.rs:21]: White space at the end of the line.

File: run-tests.py [hunk 2]
```diff
@@ -171,11 +171,13 @@ def __iter__(self):
             "missing-patch-comment",
             [
                 "missing-comment",
+                "comment-after-newline",
             ],
             [
                 "general-comment",
                 "comment-above",
                 "comment-within",
+                "comment-inline",
```
Comment (jtojnar) [inline run-tests.py:180]: This is out of order.

**Review by jtojnar — COMMENTED** [review]
Two small nits, looks great otherwise.

**Comment by jtojnar** [comment]
Thanks.

**Pull request merged**